modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm

```diff
@@ -58,6 +58,7 @@
 	name = "Tactical Boots"
 	item_path = /obj/item/clothing/shoes/combat/rax
 	//ckeywhitelist = list("raxraus")
+	restricted_roles = list(JOB_BLUESHIELD, JOB_HEAD_OF_SECURITY, JOB_SECURITY_OFFICER, JOB_WARDEN, JOB_DETECTIVE, JOB_SECURITY_MEDIC, JOB_CORRECTIONS_OFFICER) //BUBBER EDIT
```

Comment:
This has been done modularly in #734

Either remove this edit or make it modular

Reply:
Also add an "Alternative to #734" into the pr description to let maintainers know the close the other if this gets through

Reply:
Oh mb I didn't even know there was one done already i'll make it modular rq, and add that alternative comment.

Reply:
Should be all good now, I did this at like 2 AM so that probably helps explain why it sucked so hard besides me naturally sucking hard at this.